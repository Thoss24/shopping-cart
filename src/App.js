import React, { useState} from "react";

function App() {

  const [test, setTest] = useState("Test")

  const testButton = () => {
    setTest("Testing")
  }

  return (
 <div>
  <h1>Hello from App</h1>
  <button onClick={testButton}></button>
 </div>
  );
}

export default App;
