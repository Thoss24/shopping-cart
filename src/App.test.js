import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import App from './App';

// test takes the name of the test and a callback function as it's two arguments
describe("Testing app component", () => {
  it("Test", () => {

    const {container} = render(<App />);
    expect(container).toMatchSnapshot();
  });

})