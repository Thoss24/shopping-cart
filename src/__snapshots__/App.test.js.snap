// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div>
    <h1>
      Hello from App
    </h1>
  </div>
</div>
`;

exports[`Testing app component Test 1`] = `
<div>
  <div>
    <h1>
      Hello from App
    </h1>
    <button />
  </div>
</div>
`;
